docroot/sites/admissions.uiowa.edu/modules/admissions_core/src/Plugin/Block/AOSRequirements.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\admissions_core\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * An Area of Study Requirements Heading block.
+ *
+ * @Block(
+ *   id = "aosrequirements_block",
+ *   admin_label = @Translation("Area of Study Requirements Heading Block"),
+ *   category = @Translation("Area of Study")
+ * )
+ */
+class AOSRequirements extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return ['label_display' => FALSE];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $markup = '<div class="element--bold-intro text-align-center">
+            New Headline
```

Comment:
Is this supposed to say New Headline?

Reply:
Yes, they don't know what they want it to say yet.